ls-ui-react-app/src/pages/LatticeViewPage.css

```diff
@@ -0,0 +1,54 @@
+/* lattice view styles */
+
+/*html, body{
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  white-space: nowrap;
+  overflow: hidden;
+}*/
+
+.lattice-card{
+  border: 2px solid black;
+  border-radius: 15px;
+  width:200px;
+  background-color:#ffdbb3 !important;
+}
+.center {
+  text-align:center;
+}
+
+#lattice-view-output {
+  /*Hacky way to make sure footer stays at the bottom when there isn't much page content*/
+  min-height: 90vh;
+}
+
+#toolbar{
+    /* Make it independent of zoom */
+  height: 10%;
+  line-height: 10vh;
+  font-size: 5vh;
+  /* Keep it above the lattice */
+  z-index: 100;
+  position: relative;
+  /* style */
+  background-color: lavender;
+  border-bottom: solid darkgrey 1.5vh;
+}
+#draggable-container{
+  font-size: 0;
+  z-index: 10;
+}
+
+.lattice-cell{
+  border: 0.5px solid #a2abae;
+  height: 50pt;
+  width: 50pt;
+  display: inline-block;
+  position: relative;
+  font-size:9px;
+}
```

Comment:
This doesn't work because `<CellViewer />` doesn't contain the `<div className="lattice-cell">` anymore. It might be the case that the `<div className="lattice-cell-inside">` needs to be wrapped in another div with class `lattice-cell`.

Perhaps these styles are best added inline with Emotion. Either way I suspect you still need a new div in  `<CellViewer />`  that wraps  `<div className="lattice-cell-inside">`